### ISSUE TITLE
Packed executable is broken if project path contains a space

### ISSUE THREAD

**Issue by gsmaverick** [xerial/sbt-pack#76]
If the project path being packed contains a space (on OS X at least) the packed executable fails to build properly and gives this error when run:

```
$ ./target/pack/bin/forge
Error: Could not find or load main class space.inkling-forge.target.pack.icon-mac.png
```

where the path to the project is `/Volumes/test space/inkling-forge`. This very same project packs fine when the project path does not contain a space (`/Volumes/Source/inkling-forge`).

**Comment by xerial** [comment]
Some quotation is missing in the script. OK. Let me check

**Comment by xerial** [comment]
I'll release 0.7.4 with this fix.

**Comment by gsmaverick** [comment]
Unfortunately this didn't fix it for us. There were some problems with the paths for the executed command: https://github.com/xerial/sbt-pack/blob/develop/src/main/templates/launch.mustache#L134.

When I changed both the paths to `-cp` and `-Dprog.home` to be in quotes it started working.

**Comment by xerial** [comment]
Thanks for the report. OK. I'll quote them

**Comment by xerial** [comment]
@gsmaverick 
Could you show me how did you quote them?

**Comment by gsmaverick** [comment]
Sorry for the delay on this, after the bash file was generated I had to make sure the path specified here: https://github.com/xerial/sbt-pack/blob/develop/src/main/templates/launch.mustache#L143 was fully quoted (as well as the one on line 138 I believe).

**Comment by xerial** [comment]
For example, this line is already have double quotations:
```
  -Dprog.home="${PROG_HOME}"
```
I would like to know how did you *full-quote* this?

**Comment by gsmaverick** [comment]
I edited the generated script in a text editor? I'm not entirely sure I'm understanding your question though...

**Comment by xerial** [comment]
@gsmaverick 
I want to see your concrete example after editing the above line :)

**Comment by xerial** [comment]
Something like this works?
```
-Dprog.home='"${PROG_HOME}"' 
```

**Comment by xerial** [comment]
Released sbt-pack-0.7.5 with this fix. Thanks for reporting!

**Comment by gsmaverick** [comment]
Sorry I forgot to reply to your earlier request. 0.7.5 fixed the issue though, thank you!
